runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/kotlin/kotlinClasspathUtils.kt

```diff
@@ -25,6 +25,7 @@ internal fun Project.classpathOf(sourceSet: KotlinSourceSet): FileCollection {
 private fun Project.compileClasspathOf(compilation: KotlinCompilation): FileCollection {
     if (compilation.target.isAndroidTarget()) {
         // This is a workaround for https://youtrack.jetbrains.com/issue/KT-33893
+        @Suppress("DEPRECATION") // for compatibility
         return compilation.compileKotlinTask.cast<KotlinCompile>().classpath
```

Comment:
Is there any replacement for this? If yes, can we migrate to it or is that too much work?

Reply:
A replacement `KotlinCompile.libraries` does not work in old KGP version since it is absent.